src/olympia/amo/tests/test_helpers.py

```diff
@@ -240,6 +240,18 @@ def test_urlparams_returns_safe_string():
     s = render('{{ "https://foo.com/"|urlparams(param="%AAA") }}', {})
     assert s == 'https://foo.com/?param=%AAA'
 
+    string = render(
+        '{{ unsafe_url|urlparams }}',
+        {
+            'unsafe_url': "http://url.with?foo=<script>alert('awesome')</script>"
+            "&baa=that"
+        },
+    )
+    assert string == (
+        'http://url.with?foo=%3Cscript%3Ealert%28%27awesome%27%29%3C%2Fscript%3E'
+        '&baa=that'
+    )
+
```

Comment:
Could you also add a test adding unsafe params to a normal url ?

Reply:
https://github.com/mozilla/addons-server/pull/16614/commits/02aa4ceee7ce14aa3e474883585421ba28f02338